packages/jira-adapter/src/filters/automation/handle_template_expressions.ts

```diff
@@ -0,0 +1,142 @@
+/*
+*                      Copyright 2022 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+import {
+  Change, Element, getChangeData, InstanceElement, isInstanceElement,
+  ReferenceExpression, TemplateExpression, TemplatePart, Values,
+} from '@salto-io/adapter-api'
+import { extractTemplate, replaceTemplatesWithValues, resolveTemplates } from '@salto-io/adapter-utils'
+import { logger } from '@salto-io/logging'
+import { collections } from '@salto-io/lowerdash'
+import _ from 'lodash'
+import { FilterCreator } from '../../filter'
+
+const { awu } = collections.asynciterable
+const log = logger(module)
+const REFERENCE_MARKER_REGEX = /(\{\{)(.+?)(}})/
+
+
+type Component = Record<string, Values>
+const simpleIssueRegex = new RegExp(/issue\.[a-zA-Z ]+(?: |$)/, 'g')
+const issueFieldNameToTypeMapping: Record<string, string> = {
+  key: 'Key',
+  project: 'Project',
+  status: 'Status',
+  summary: 'Summary',
+  versions: 'Affects versions',
+  resolution: 'Resolution',
+  updated: 'Updated',
+  'Story Points': 'Story Points',
+  'Story Points estimate': 'Story Points estimate',
+  created: 'Created',
+  duedate: 'Due Date',
+}
+
+const issueTypeNameToFieldMapping = Object.fromEntries(
+  Object.entries(issueFieldNameToTypeMapping).map(([k, v]) => [v, k])
+)
+
+const getAutomations = (instances: InstanceElement[]): InstanceElement[] =>
+  instances.filter(instance => instance.elemID.typeName === 'Automation')
+
+// This function receives a string that contains issue references and replaces
+// it with salto style templates.
+const stringToTemplate = (referenceSt: string,
+  fieldsByName: Record<string, InstanceElement>): TemplateExpression | string => {
```

Comment:
readability nit
```suggestion
const stringToTemplate = (
  referenceSt: string,
  fieldsByName: Record<string, InstanceElement>,
): TemplateExpression | string => {
```
or
```suggestion
const stringToTemplate = (
  referenceSt: string, fieldsByName: Record<string, InstanceElement>,
): TemplateExpression | string => {
```